docs/blog/newsletter-august-2020.rst

```diff
@@ -0,0 +1,85 @@
+.. post:: August 02, 2020
+   :tags: newsletter
+
+#######################################
+Write the Docs Newsletter – August 2020
+#######################################
+
+Dear documentarians of the world - good morning, afternoon, or evening! It’s Beth here, writing to you on an absolutely sweltering day in Switzerland. Not going to lie, 36 degrees C in the home office is rather a challenge; on the plus side, at home a mid-afternoon siesta becomes a very viable option… 
+
+The `virtual Portland conference </conf/portland/2020>`__ is fast approaching, on 9-11 August. Tickets are `on the verge of selling out </conf/portland/2020/news/tickets-selling-out/>`__, so if you’ve been sitting on the fence about whether to go, get a move on! `Here’s the schedule </conf/portland/2020/news/schedule/>`__ if it’ll help you make up your mind.
+
+And if Portland isn’t in a convenient time zone for you, fear not! because the `Australia and India virtual conference </conf/australia/2020/>`__ has just announced both dates (3-4 December 2020) and a `call for proposals </conf/australia/2020/news/cfp-open/>`__. We cannot wait to hear from documentarians far and wide - get those talk proposals submitted by 31st August.
+
+Next month, the newsletter team is on our summer break - so enjoy this month’s articles, and I’ll look forward to popping back into your inbox again come October. Stay well!
+
+---------------------------------
+To git or not to git docs as code
+---------------------------------
+
+Everybody's asking about a docs-as-code workflow and toolchain these days - one recent question about adopting it for a team that's unfamiliar with Git sparked a useful discussion about this potentially rather painful learning curve. Here's the advice from folks who've brought teams up to speed with Git:
+
+* It's crucial to have at least one Git expert available to help troubleshoot, both as people are learning for the first time and as everyone on the team (no matter how proficient) will encounter new and unexpected issues.
```

Comment:
this needs rewording but I don't have enough coffee yet

> both as people are learning for the first time and as everyone on the team (no matter how proficient) will encounter new and unexpected issues

Reply:
Yeah, I wasn't quite sure how to put this one either. Have rephrased.